octoprint_mrbeam/static/updates/update_info.json

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
this should be removed

Reply:
We need to make sure that the SW always shows the currently installed versions of the SW components - even if we do not have a locally downloaded version and no internet connection.

Reply:
ok, it will fallback to mrbeamplugin file if there is no from server or localy